components/portal_players/commons/portal_players.lua

```diff
@@ -284,6 +284,18 @@ function PortalPlayers:row(player, isPlayer)
 	return row
 end
 
+---@param status string?
+---@param banned string?
+---@return string?
+function PortalPlayers._getStatusBackground(status, banned)
+	if Logic.isEmpty(status) then
+		status = Logic.emptyOr(Logic.readBoolOrNil(banned), Logic.isNotEmpty(banned))
+			and 'banned' or nil
+	end
```

Comment:
iirc, status will be `inactive`, not empty when banned is set, so this won't ever work
```suggestion
	if Logic.readBoolOrNil(banned) then
		status = 'banned'
	end
```

Reply:
banned might be non bool text
and imo status should have precidence unless inactive
```suggestion
	if status == STATUS_INACTIVE then
		status = Logic.emptyOr(Logic.readBoolOrNil(banned), Logic.isNotEmpty(banned))
			and 'banned' or nil
	end
```